templates/taxi/car_detail.html

```diff
@@ -8,5 +8,10 @@ <h1>Drivers</h1>
   {% for driver in car.drivers.all %}
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
+  <br>
+    <h4><a href="{% url 'taxi:car-update' car.id %}">Update</a>
+    <br><br>
+    <a href="{% url 'taxi:car-delete' car.id %}">Delete</a>
+    </h4>
```

Comment:
The `<h4>` tag is used to wrap both the 'Update' and 'Delete' links, which is not semantically correct as these are not headings. Consider using a `<div>` or another appropriate container element instead.